pkg/api/frontendsettings.go

```diff
@@ -35,6 +35,13 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		return nil, err
 	}
 
+	pluginExtensions := map[string]*plugins.Extensions{}
+	for _, app := range enabledPlugins[plugins.App] {
+		if app.Extensions != nil {
+			pluginExtensions[app.ID] = app.Extensions
+		}
+	}
+
```

Comment:
Question: maybe it would make sense to add a quick test for this in `frontendsettings_test.go`, wdyt?

Reply:
Yes, that is part of the plan :)